.github/workflows/unit-tests.yml

```
@@ -0,0 +1,77 @@
+name: Ci suite
+on: [push, pull_request]
+jobs:
+    symfony:
+        name: PHPUnit tests (PHP ${{ matrix.php-versions }})
+        runs-on: ubuntu-latest
+
+        # Docs: https://docs.github.com/en/actions/using-containerized-services
+        services:
+            postgres:
+                image: postgres:latest
+                env:
+                    POSTGRES_USER: postgres
+                    POSTGRES_PASSWORD: postgres
+                    POSTGRES_DB: postgres
+                ports:
+                    - 5432/tcp
+                options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 3
```

```suggestion
                options: --health-cmd pg_isready --health-interval 1s --health-timeout 3s --health-retries 3
```

no need to wait that long